internal/ui/ui.go

```diff
@@ -336,14 +362,27 @@ func footer(width int, time string, groupSelectedName string) string {
 		groupSelectedName = groupSelectedName[:12]
 	}
 
+	// Get display name for current sort
+	sortDisplayName := "change"
+	switch currentSort {
+	case "alpha":
+		sortDisplayName = "alpha"
+	case "value":
+		sortDisplayName = "value"
+	case "user":
+		sortDisplayName = "user"
+	}
+
+	helpText := " q: exit ↑: scroll up ↓: scroll down s: change sort (" + sortDisplayName + ") ⭾: change group"
+
 	return grid.Render(grid.Grid{
 		Rows: []grid.Row{
 			{
 				Width: width,
 				Cells: []grid.Cell{
 					{Text: styleLogo(" ticker "), Width: 8},
 					{Text: styleGroup(" " + groupSelectedName + " "), Width: len(groupSelectedName) + 2, VisibleMinWidth: 95},
-					{Text: styleHelp(" q: exit ↑: scroll up ↓: scroll down ⭾: change group"), Width: 52},
+					{Text: styleHelp(helpText), Width: len(helpText)},
```

Comment:
The overall line width should be less than 80 and handle progressive display otherwise. I'd suggest using `VisibleMinWidth` to hide parts of this section based on screen width